benchmark/match-parameter.lua

```diff
@@ -1,10 +1,10 @@
 local radix = require("resty.radixtree")
-local route_count = 1000 * 100
+local route_count = 1000 * 10
```

Comment:
`route_count` is 1000 * 100 in both `match-prefix.lua` and `match-static.lua`, I think it should be the same here(for control variates)?

Reply:
nice, fixed